### PR TITLE
Readme added.

### PR DESCRIPTION
**Pull request by filipstachura**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,74 @@
...
+<a href="http://appsilondatascience.com/demos">Live demo</a>
+</p>
+-->
+
+Source code
+-----------
+
+This library source code can be found on [Appsilon Data Science's](http://appsilondatascience.com) Github: <br> <https://github.com/Appsilon/shiny.i18n/>
+
+<script>
```
Comment (dokato) [inline README.md]: Are you sure that those scripts should be here? for example in `sc` README I cannot find it: https://raw.githubusercontent.com/Appsilon/shiny.collections/master/README.md

File: README.md [hunk 2]
```diff
@@ -1 +1,74 @@
...
+document.write('<div class="logo"><a href="http://appsilondatascience.com"><img alt="Appsilon Data Science" src="https://cdn.rawgit.com/Appsilon/website-cdn/gh-pages/logo-white.png"/></a></div>')
```
Comment (dokato) [inline README.md]: White logo looks nice on documentation website, but on github white page looks like sth is missing.

Comment (filipstachura) [reply README.md]: Yeah, I know - probably we need to change that depending on the target. Let me try to fix that.

File: README.md (hunk 1 above)
Comment (filipstachura) [reply README.md]: Because this are in Rmd files: https://raw.githubusercontent.com/Appsilon/shiny.collections/master/README.Rmd

Comment (filipstachura) [reply README.md]: Oh, I know what you mean! This is fixed now with: https://github.com/Appsilon/shiny.i18n/pull/5

**Pull request merged**